### PR TITLE
Add Harmony docs

### PR DESCRIPTION
**Pull request by akegaviar**

### CODE REVIEW

**Comment by akegaviar** [comment]
To be checked with our endpoints + features and then convert from WIP to a full PR.

**Comment by akegaviar** [comment]
@0xpranjal please check the entire tools section and make sure all the tools and libraries listed there work with our Harmony node endpoints (when these become available) both for basic auth and key-protected endpoint types.

File: docs/operations/harmony/tools.md [hunk 1]
```diff
@@ -0,0 +1,881 @@
...
+      * Shard 0: `1666600000`
+      * Shard 1: `1666600001`
+      * Shard 2: `1666600002`
+      * Shard 3: `1666600003`
+    * Testnet:
+      * Shard 0: `1666700000`
+      * Shard 1: `1666700001`
+      * Shard 2: `1666700002`
+      * Shard 3: `1666700003`
+
```
Comment (0xpranjal) [inline docs/operations/harmony/tools.md:200]: Can we add one step before Save: "Write ONE in the Symbol feild  and  https://explorer.harmony.one as the Block Explorer URL"

File: docs/operations/harmony/tools.md [hunk 2]
```diff
@@ -0,0 +1,881 @@
...
+        network_id: "*"
+    },
+   }
+  }
+};
+```
+
+</template>
+</CodeSwitcher>
+
```
Comment (0xpranjal) [inline docs/operations/harmony/tools.md:253]: Can also link Truffle Harmony box: https://trufflesuite.com/boxes/harmony-box/

Comment (0xpranjal) [inline docs/operations/harmony/tools.md:253]: Add final step to compile and deploy:

truffle compile
truffle deploy --network testnet

File: docs/operations/harmony/tools.md [hunk 3]
```diff
@@ -0,0 +1,881 @@
...
+``` js
+const HDWalletProvider = require("@truffle/hdwallet-provider");
+const mnemonic = 'pattern enroll upgrade ...';
+...
+module.exports = {
+ networks: {
+    chainstack: {
+        provider: () => new HDWalletProvider(mnemonic, "https://nd-123-456-789.p2pify.com/3c6e0b8a9c15224a8228b9a98ca1531d"),
+        network_id: "*"
```
Comment (0xpranjal) [inline docs/operations/harmony/tools.md:226]: Replace * -> "1666700000"

**Review by 0xpranjal — APPROVED** [review]
Suggested few additional tools to add specific to Harmony. Also, verified integration of all the tools with Harmony docs, looks good to me.

File: docs/operations/harmony/modes.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+  - name: keywords
+    content: harmony geth one node full archive difference prune
+---
+
+# Modes
+
+Chainstack supports deploying a Harmony node on the mainnet in the following modes:
+
+* Full node — a node that stores full blockchain data.
+* Archive node — a node that stores full blockchain data and an archive of historical states.
```
Comment (easeev) [inline docs/operations/harmony/modes.md:14]: We don’t support archive nodes for now

File: docs/platform/join-a-public-network.md [hunk 5]
```diff
@@ -99,6 +99,21 @@ The node status will change from **Pending** to **Running** once deployed.
 
 The node status will change from **Pending** to **Running** once deployed.
 
+## Join a Harmony network
+
+1. Select a [public chain project](/glossary/public-chain-project) and click **Get started** or **Join network**.
+1. Under **Blockchain protocol**, select **Harmony**.
+1. Under **Blockchain network**, select **Mainnet** or **Testnet**. Click **Next**.
+1. Under **Type**, select whether to run a [shared](/glossary/shared-node) or a [dedicated](/glossary/dedicated-node) node.
+1. Under **Mode**, select whether to run a full node or an archive node. See [Modes](/operations/fantom/modes).
```
Comment (easeev) [inline docs/platform/join-a-public-network.md:108]: See above

File: docs/operations/harmony/modes.md (hunk 4 above)
Comment (akegaviar) [reply docs/operations/harmony/modes.md:22]: Fixed as discussed withe product management and engineering teams.

File: docs/platform/join-a-public-network.md (hunk 5 above)
Comment (akegaviar) [reply docs/platform/join-a-public-network.md]: fixed

**Pull request merged**